### PR TITLE
initialisation of chrome options and change default value of  guzzle client option to false

### PR DESCRIPTION
**Pull request by suciista09**
changes 
1. initialisation of chrome options
2. set default value of verify to false (guzzle client)

### CODE REVIEW

File: src/Api/ApiClientBuilder.php [hunk 1]
```diff
@@ -93,6 +93,7 @@ public function build()
         }
 
         $guzzleClient->setDefaultOption('exceptions', $this->httpExceptions);
+        $guzzleClient->setDefaultOption('verify',false);
```
Comment (pproenca) [inline src/Api/ApiClientBuilder.php]: This option should be retrieved from the settings class and if it's empty, then default to `false`

**Review by pproenca — CHANGES_REQUESTED** [review]

File: src/Api/ApiClientBuilder.php (hunk 1 above)
Comment (suciista09) [reply src/Api/ApiClientBuilder.php]: what is setting class? is it Json file configuration?

Comment (pproenca) [reply src/Api/ApiClientBuilder.php]: You can add a method in the `ApiClientBuilder` that set's whether the verify option is true or false.

Then in the `Athena.php` file, where the `ApiClientBuilder` is called, you pass the pass the value to the builder using the settings class.

Example:
https://github.com/athena-oss/plugin-php/blob/master/src/Athena.php#L278

**Comment by suciista09** [comment]
@pproenca Hi Pedro please check my new commit

**Pull request merged**